pkg/cli/userfiletable_test.go

```diff
@@ -98,6 +249,126 @@ func checkUserFileContent(
 	require.True(t, bytes.Equal(got, expectedContent))
 }
 
+func TestUserFileUploadRecursive(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+
+	c := NewCLITest(TestCLIParams{T: t})
+	defer c.Cleanup()
+	c.omitArgs = true
+
+	tmpDir, cleanFn := testutils.TempDir(t)
```

Comment:
any chance we can pull this directory setup to a helper? Looks like it is almost the same between the example test and this test.

Maybe a helper that returns the testDir and a cleanup function? Ignore me if I'm missing something more fundamental that prevents use from doing this.